### ISSUE TITLE
How 同

### ISSUE THREAD

**Issue by Shaoqing26** [ApolloAuto/apollo#12623]
We appreciate you go through Apollo documentations and search previous issues before creating an new one. If neither of the sources helped you with your issues, please report the issue using the following form. Please note missing info can delay the response time.

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**:
- **Apollo installed from (source or binary)**:
- **Apollo version (3.5, 5.0, 5.5, 6.0)**:
- **Output of `apollo.sh config` if on `master` branch**:

### Steps to reproduce the issue:
- **Please use bullet points and include as much details as possible**:

**Comment by jeroldchen** [comment]
@Shaoqing26 Yes, we provide an online service for training PointPillars model using your own data. For the details, please check 
[https://github.com/ApolloAuto/apollo/blob/master/docs/Apollo_Fuel/Perception_Lidar_Model_Training/README.md](url) .

**Comment by Shaoqing26** [comment]
> @Shaoqing26 Yes, we provide an online service for training PointPillars model using your own data. For the details, please check
> [https://github.com/ApolloAuto/apollo/blob/master/docs/Apollo_Fuel/Perception_Lidar_Model_Training/README.md](url) .

 Can u prove offlin python version for tranning? and I have no camera data fusion the lidar How  can i training with only lidar data? .

**Comment by panfengsu** [comment]
@Shaoqing26 now we only provide cloud service of training PointPillars model.We will support more training data format later.Can you tell your own data format, we will support it as far as possible.Thank you for supporting Apollo!

**Comment by Shaoqing26** [comment]
> @Shaoqing26 now we only provide cloud service of training PointPillars model.We will support more training data format later.Can you tell your own data format, we will support it as far as possible.Thank you for supporting Apollo!

Fine,actually,I dont care the format of the data,because i can convert my lidar data to kitti format to traning,but i have only lidar data.,i dont have camera information,Pointpillar is a point based Network,Why dont you separate it to  have only lidar formation.

Others,i want to use the network in apollo with 16 chanel or 32 chanel and other chanels but not 128,so  i have to train my our model,but apollo only support online trainning with BOS and apollo fuel platform, Neither of them is free.

Since apollo is open source, why can't  provide an offline training version for us, let us use it freely.

**Comment by panfengsu** [comment]
@Shaoqing26 we have see your need,autually apollo-fuel is a Saas service，not needing user to build the environment including software and GPU.So it is more convenient to train models.We will seriously consider your suggestion.

**Comment by Shaoqing26** [comment]
> @Shaoqing26 we have see your need,autually apollo-fuel is a Saas service，not needing user to build the environment including software and GPU.So it is more convenient to train models.We will seriously consider your suggestion.

OK.Thanks.
One more question,Do i have to need both of lidar and camera information to training model in apollo-fuel,in other words,How can i get nomal model with only lidar data ?

**Comment by panfengsu** [comment]
@Shaoqing26 autually,camera information is not needed,we will optimization it quickly.But I need to know how you label your lidar data,and it is in camera coordinates or lidar coordinates.We are happy that you put forward your specific needs to help us optimize this service,and we seriously consider your suggestion to provide an offine training version too.

**Comment by Shaoqing26** [comment]
> @Shaoqing26 autually,camera information is not needed,we will optimization it quickly.But I need to know how you label your lidar data,and it is in camera coordinates or lidar coordinates.We are happy that you put forward your specific needs to help us optimize this service,and we seriously consider your suggestion to provide an offine training version too.

Thanks.
I know that pointpillar is a pure pointcloud network.
Most of pointcloud Detection NetWork were use kitii  data to tranning and  pub papers,and kitti data has both camera and lidar data, it  Project each other in the network.\
We can just label object in pointcloud with: 
location formation:center of object x,y,z,height,width,lenth or label 8 points of cube.
rotation information:pitch.row.roll
and others like type /truncate  can just like kitti.

I think location information like 8 points of cude is enough. We can calculate other informations.
JUST dont use camera data to project to each other in network.

**Comment by panfengsu** [comment]
@Shaoqing26 you are right,but you must convert your location information and rotation information to lidar coordinate.kitti annotation information is in camera coordinates,so it need camera image and calibrate information.

**Comment by Shaoqing26** [comment]
> @Shaoqing26 you are right,but you must convert your location information and rotation information to lidar coordinate.kitti annotation information is in camera coordinates,so it need camera image and calibrate information.

Yes i know that,the kitti information is in camera coordinata and transform in network, but i dont want anny camera information in network. Because Pointpillar itself does not need picture information, just take the pure pointcloud into the net,and get the result.

Proposal:
1) Provide offline tranning version for apollo funs
2)separation the pure lidar data and camera data,take the pure pointcloud into network.

I think thers has lots of apollo-funs has same confuse like me.

Wish apollo get better and better.
